src/components/Common/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { css } from 'styled-components/macro'
+
+export const ScrollBarStyles = css<{ isHorizontalScroll?: boolean }>`
+  // Firefox scrollbar styling
+  scrollbar-width: thin;
+  scrollbar-color: ${({ theme }) => `${theme.backgroundOutline} transparent`};
+  overflow-y: scroll;
+  height: 100%;
+
+  // safari and chrome scrollbar styling
+  ::-webkit-scrollbar {
+    background: transparent;
+
+    // Set height for horizontal scrolls
+    ${({ isHorizontalScroll }) => {
```

Comment:
I reuse this a good amount, not sure where to put it.

Reply:
nit: make `isHorizontalScroll` transient https://styled-components.com/docs/api#transient-props